va/http.go

```diff
@@ -330,36 +338,28 @@ func (va *ValidationAuthorityImpl) fetchHTTPSimple(
 	return body, records, nil
 }
 
-// fallbackErr returns true only for:
-//   1. net.OpError instances
-//   2. net.Error instances
-//   3. url.Error instances wrapping one of the above
-// fallbackErr returns false for all other errors. By policy we're only
-// interested in retrying requests that were made to an IPv6 address that
-// generated a network error.
+// fallbackErr returns true only for net.OpError instances where the op is equal
+// to "dial", or url.Error instances wrapping such an error. fallbackErr returns
+// false for all other errors. By policy we're only interested in retrying
```

Comment:
This last sentence is a little confusing because it doesn't put the part relevant to this function up front. I might say: "By policy, only dial errors (not read or write errors) are eligible for fallback from an IPv6 to an IPv4 address."

Reply:
That reads a lot better, thanks for the suggestion :+1: Adjusted in 272a582